test/unit/tech/tech.test.js

```diff
@@ -195,6 +196,23 @@ test('should handle unsupported sources with the source handler API', function()
   ok(usedNative, 'native source handler was used when an unsupported source was set');
 });
 
+test('should allow custom error events to be set', function() {
+  let tech = new Tech(), errors = [];
```

Comment:
The style is now one var per let/const keyword, per line. https://github.com/airbnb/javascript#13.2